package.json

```diff
@@ -60,6 +62,7 @@
         "@types/jasmine": "~5.1.5",
         "@types/node": "^22.10.5",
         "autoprefixer": "^10.4.20",
+        "biome": "^0.3.3",
```

Comment:
This `biome` package is the wrong one. I fixed the upstream project (ng-starter-no-semi) last day and the git commit issue should be fixed without installing this library now. So please remove this and try again.